test/y2ftp/clients/ftp_server_test.rb

```diff
@@ -22,9 +22,11 @@
 require_relative "../../spec_helper.rb"
 require "y2ftp/clients/ftp_server"
 
-stub_module("Users")
-
 describe Y2Ftp::Clients::FtpServer do
+  before do
+    stub_module("FtpServer")
```

Comment:
ugh, this looks strange for me, why you stub your own module?

Reply:
He is stubbing the module, not the client (which is what this test is about).

Reply:
Here I am testing a client, and I am stubbing the module used by the client. I don't want a real module object, only a stub object with a mocked method. The real module object is irrelevant for this test.

Reply:
@imobachgs @joseivanlopez that is definitively over-mocking for me and also abusing stub_module to something that is not designed for. Stub module is designed to stub modules living outside of package to prevent build time dependencies. So it weakens test power, which is acceptable for faster build. But here it is not not outside of module. Which is what looking strange for me.